packages/tools/pie-icons-webc/build.js

```diff
@@ -44,6 +44,12 @@ export class ${name} extends LitElement implements IconProps {
             width: var(--btn-icon-size);
             height: var(--btn-icon-size);
         }
+
+        :host-context(pie-tag) svg {
```

Comment:
You might need to change this a bit as host-context doesn't work in firefox and safari. Unless you got this working somehow? Here's a pr i made for ref: https://github.com/justeattakeaway/pie/pull/1107/files#diff-beb4382d1fd71394d71c0f0f07f2e5111af3e72f86d94ce723a1a03b48251af9R41

Reply:
Hmm, does your change `:host svg {` mean that all the svgs will have `var(--btn-icon-size, 24px)`? It doesn't feel right. I think we need to find a way to dictate the size prop value to the icon slot as different components that use icons will need different icon sizes

Reply:
I checked link component with an icon in Firefox after your change and not it looks off 😞 :

<img width="156" alt="Screenshot 2023-12-21 at 10 49 26" src="https://github.com/justeattakeaway/pie/assets/19548183/a96b22cb-a846-4abe-8ddf-83ad6c38f92c">